python/ray/actor.py

```diff
@@ -1010,6 +1010,15 @@ def create(cls, modified_class, actor_creation_function_descriptor):
 
         # Update cache.
         cls._cache[actor_creation_function_descriptor] = self
+
+        # FIXME(Qiaolin-Yu): Check the tensor_transport option after #54256 is merged.
+        # Register a custom serializer for torch.Tensor. This allows torch.Tensors
+        # to use the created collective for communication between actors, instead of
+        # the normal serialize -> object store -> deserialize codepath.
+        from ray.experimental.channel.torch_tensor_type import TorchTensorType
```

Comment:
We should try to fix this as part of this PR since this is a breaking change (torch.Tensor serialization will silently change).

You can take the code from #54256 and we can reserve the merge conflict later.